extension/devtools/devtools_scraper_panel.html

```diff
@@ -22,7 +22,9 @@
     <script src="../scripts/Selector/SelectorGroup.js"></script>
     <script src="../scripts/Selector/SelectorText.js"></script>
     <script src="../scripts/Selector/SelectorValue.js"></script>
```

Comment:
SelectorValue is name that hard to
to distinguish from ConstantValue. I suggest to rename it to InputValue.